GiniVision/Classes/CameraViewController.swift

```diff
@@ -84,7 +93,10 @@ public typealias CameraErrorBlock = (_ error: CameraError) -> ()
         return UIImageNamedPreferred(named: "cameraCaptureButtonActive")
     }
     fileprivate var cameraOverlayImage: UIImage? {
-        return UIImageNamedPreferred(named: "cameraOverlay")
+        guard let image = UIImageNamedPreferred(named: "cameraOverlay"), let cgImage = image.cgImage else{
```

Comment:
Isn't the cameraOverlay image too small to scale it up to iPad sizes? Doesn't it get blurry?

Ideally, I'd like to remove it and add the rectangle programmatically. We can copy the code from GiniSwitch.

The problem is that the image is also customisable for clients so we need a major version update to fix it. @a-szotyori , do you think  we can do it now or in the near future?